### PR TITLE
[HLT] Apply code-checks/format

### PR DESCRIPTION
**Pull request by smuzaffar**
code-checks/format applied on full cmssw with extra misc-definitions-in-headers clang-tidy check

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

**Comment by smuzaffar** [comment]
@cms-sw/hlt-l2 can you please review this?

File: HLTrigger/Muon/plugins/HLTTriMuonIsolation.h [hunk 1]
```diff
@@ -92,9 +92,9 @@ HLTTriMuonIsolation::HLTTriMuonIsolation(const edm::ParameterSet& iConfig)
   produces<reco::CompositeCandidateCollection>("SelectedTaus");
 }
 
-HLTTriMuonIsolation::~HLTTriMuonIsolation() {}
```
Comment (fwyzard) [inline HLTrigger/Muon/plugins/HLTTriMuonIsolation.h:95]: this should actually be defined `= default;` in the class body

File: HLTrigger/Muon/plugins/HLTTriMuonIsolation.h [hunk 2]
```diff
@@ -64,7 +64,7 @@ class HLTTriMuonIsolation : public edm::global::EDProducer<> {
   const bool EnableAbsIso_;
 };
 
-HLTTriMuonIsolation::HLTTriMuonIsolation(const edm::ParameterSet& iConfig)
+inline HLTTriMuonIsolation::HLTTriMuonIsolation(const edm::ParameterSet& iConfig)
```
Comment (fwyzard) [inline HLTrigger/Muon/plugins/HLTTriMuonIsolation.h:67]: do we need the `inline` also in the method declaration inside the class body ?

Comment (smuzaffar) [reply HLTrigger/Muon/plugins/HLTTriMuonIsolation.h:67]: there is no need for that.

**Comment by fwyzard** [comment]
OK, I guess this can go in as is - and eventually we can make the destructor change separately.

**Comment by fwyzard** [comment]
+hlt

File: HLTrigger/Muon/plugins/HLTTriMuonIsolation.h (hunk 1 above)
Comment (smuzaffar) [reply HLTrigger/Muon/plugins/HLTTriMuonIsolation.h:95]: done

**Comment by smuzaffar** [comment]
sorrt @fwyzard , I did not see your `+hlt`, I have update the PR to include the default destructor change.

**Comment by fwyzard** [comment]
please test

**Comment by fwyzard** [comment]
+hlt

**Comment by qliphy** [comment]
+1

**Pull request merged**